tensorizer/serialization.py

```diff
@@ -19,15 +20,15 @@
 import tensorizer.stream_io as stream_io
 import tensorizer.utils as utils
 import mmap
-# Python doesn't know about this.
-MADV_PAGEOUT = 21
 import numpy
 import struct
+import time
 import torch
 import typing
 import logging
 import tempfile
 import regex
```

Comment:
We should switch to the built-in module `re` instead of the third-party library `regex` since our use case doesn't require the added features in `regex`.